### PR TITLE
Bug 1496419 - Fix graphs when nudging/manually creating alerts

### PR DESCRIPTION
**Pull request by ionutgoldan**

### CODE REVIEW

**Comment by ionutgoldan** [comment]
@jmaher the nudge now works from FE too. What's left is to automatically show to tool tip popup on the new position of the nudged alert.

File: ui/js/controllers/perf/graphs.js [hunk 1]
```diff
@@ -66,13 +66,25 @@ perf.controller('GraphsCtrl', [
             const resultSetData = dataPoint.series.flotSeries.resultSetData;
             const towardsDataPoint = PhAlerts.findPushIdNeighbours(dataPoint, resultSetData, direction);
 
-            PhAlerts.nudgeAlert(dataPoint, towardsDataPoint).then(() => {
-                $scope.nudgingAlert = false;
+            PhAlerts.nudgeAlert(dataPoint, towardsDataPoint).then((alertSummaryId) => {
+                PhAlerts.getAlertSummaries({
```
[the remark this answers is not in the document]
Comment (ionutgoldan) [reply ui/js/controllers/perf/graphs.js]: @wlach Done.

File: ui/js/controllers/perf/graphs.js [hunk 2]
```diff
@@ -43,20 +43,11 @@ perf.controller('GraphsCtrl', [
...
-
-                    var alertSummary = alertSummaryData.results.find(result =>
-                        result.id === alertSummaryId);
-                    $scope.tooltipContent.alertSummary = alertSummary;
-
-                    dataPoint.series.relatedAlertSummaries = alertSummaryData.results;
-                    plotGraph();
-                });
+            PhAlerts.createAlert(dataPoint)
+            .then(alertSummaryId => refetchGraphData(alertSummaryId, dataPoint))
```
Comment (wlach) [inline ui/js/controllers/perf/graphs.js]: Can you rename `refetchGraphData` to `refreshGraphData` and put the call to `plotGraph` inside it? lgtm aside from that.

Comment (ionutgoldan) [reply ui/js/controllers/perf/graphs.js]: @wlach I updated.

File: ui/js/controllers/perf/graphs.js [hunk 3]
```diff
@@ -642,6 +648,7 @@ perf.controller('GraphsCtrl', [
                     series.relatedAlertSummaries = [];
                     var repo = $rootScope.repos.find(repo =>
                         repo.name === series.projectName);
+
```
Comment (wlach) [inline ui/js/controllers/perf/graphs.js]: Can you remove this newline before landing?

Comment (ionutgoldan) [reply ui/js/controllers/perf/graphs.js]: Sure.

**Pull request merged**